### PR TITLE
static_page 2

### PR DESCRIPTION
**Pull request by huynhduc96**
screenshot from 2018-07-24 08-51-04

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</footer>
+</div>
```
Comment (hngda01) [inline app/views/layouts/_footer.html.erb]: them dong vao cuoi file

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+              <button class="header_search_button d-flex flex-column align-items-center justify-content-center">
+                <i class="fa fa-search" aria-hidden="true"></i>
+              </button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</header>
```
Comment (hngda01) [inline app/views/layouts/_header.html.erb]: them dong

File: vendor/assets/javascripts/custom.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+							document.getElementsByClassName('milestone_counter')[i].innerHTML = signBefore + counter.value + signAfter;
+						}
+		    		});
+		    	})
+			    .addTo(ctrl);
+	    	});
+		}
+	}
+
+});
```
Comment (hngda01) [inline vendor/assets/javascripts/custom.js]: them dong cuoi

File: vendor/assets/javascripts/easing.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ *  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE
+ *  GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED 
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED 
+ * OF THE POSSIBILITY OF SUCH DAMAGE. 
+ *
+ */
```
Comment (hngda01) [inline vendor/assets/javascripts/easing.js]: them dong cuoi file

**Comment by hngda01** [comment]
ok

**Comment by tuquyet2709** [comment]
ok

**Pull request merged**